packages/aragon-cli/src/commands/apm_cmds/publish.js

```diff
@@ -369,7 +418,7 @@ exports.task = function({
 
           if (!isValid) {
             throw new Error(
-              'Version bump is not valid, you have to respect APM bumps policy. Check version upgrade rules in documentation https://hack.aragon.org/docs/aragonos-ref.html#631-version-upgrade-rules'
+              'Version bump is not valid, you have to respect APM bumps policy. Check version upgrade rules in documentation https://hack.aragon.org/docs/apm-ref#version-upgrade-rules'
```

Comment:
```suggestion
              'Version bump is not valid, you have to respect aragonPM bumps policy. Check version upgrade rules in documentation https://hack.aragon.org/docs/apm-ref#version-upgrade-rules'
```

If there are other places where we still use `APM` in a user-seen output, let's also change them to be `aragonPM` :).